test/integration/xds_integration_test.cc

```diff
@@ -126,5 +126,149 @@ TEST_P(LdsIntegrationTest, FailConfigLoad) {
                              "Didn't find a registered implementation for name: 'grewgragra'");
 }
 
+using LdsIpv4OnlyIntegrationTest = HttpProtocolIntegrationTest;
+
+// The LDS code path supports both ipv4 and ipv6. We run the tests only on ipv4 supported platform
+// because ipv4 stack naturally provides multiple loopback ip addresses.
```

Comment:
I don't think this is a good idea. Some platforms are v6 only. Please figure out how to do v6 tests and remove the getIpv4OnlyProtocolTestParams function.

Reply:
The in place update should be independent from v4 or v6 platform. In place update should work on both or work on neither. We need at least one of v4 and v6 be covered. The CI guarantee the existence of v4 right? :)

Reply:
Google internally only uses v6. We absolutely must test this feature on both v4 and v6, so please remove this function and make sure the tests work on both. I don't really understand why there should be any issue with v4 or v6. 

If you need multiple filter chains just figure out some other way of splitting them. Use TLS/SNI? Use HTTP inspector match? Something else?

Reply:
The motivation was to use as few components as possible but now give away to dual stack